<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDlg</class>
 <widget class="QDialog" name="InfoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Полная информация о ДТП</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>651</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Verdana&quot;;</string>
   </property>
   <property name="title">
    <string>Участники</string>
   </property>
   <widget class="QLabel" name="label_veh">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>561</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Транспортные средства:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_ped">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>641</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Пешеходы:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>651</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Verdana&quot;;</string>
   </property>
   <property name="title">
    <string>Дорожно-транспортное происшествие</string>
   </property>
   <widget class="QLabel" name="label_desc">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Описание ДТП:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cond">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дорожные условия:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cause">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>551</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Причина ДТП:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Время суток:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_address">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Адрес:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>510</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
